samples/pwm/main.cpp

```diff
@@ -17,7 +17,7 @@ int main() {
     // 1000000 microseconds (1 second) period
     pwm.setPeriod(1000000);
     // 50 % duty cycle
-    pwm.setDutyCycle(50);
+    pwm.setDutyCycle(70);
```

Comment:
why did this change? It should probably still be 50 considering the comment.
```suggestion
    pwm.setDutyCycle(50);
```